wiki/Tournaments/OWC/2015/en.md

```diff
@@ -26,137 +26,96 @@ The **osu! World Cup 2015** (***OWC 2015***) is a country-based osu! tournament
+| ![][flag_SE] | **Sweden** | **[Xytox](https://osu.ppy.sh/users/2229274 "Xytox")**, [Slizzer](https://osu.ppy.sh/users/809983 "Slizzer"), [SnickarN](https://osu.ppy.sh/users/3258429 "SnickarN"), [AntoN](https://osu.ppy.sh/users/2538562 "AntoN"), [Bubba](https://osu.ppy.sh/users/2330524 "Bubba"), [IVo one](https://osu.ppy.sh/users/3623465 "IVo one"), [Sebbe](https://osu.ppy.sh/users/3181965 "Sebbe"), [Xiniox](https://osu.ppy.sh/users/5233691 "Xiniox") |
+| ![][flag_TW] | **Taiwan** | **[Rucker](https://osu.ppy.sh/users/147515 "Rucker")**, [hvick225](https://osu.ppy.sh/users/50265 "hvick225"), [Small K](https://osu.ppy.sh/users/952751 "Small K"), [YuyuKo sama](https://osu.ppy.sh/users/234788 "YuyuKo sama"), [Uan](https://osu.ppy.sh/users/147623 "Uan"), [dabanlong](https://osu.ppy.sh/users/624254 "dabanlong"), [zxxzxxz](https://osu.ppy.sh/users/1646474 "zxxzxxz"), [RedLeaf](https://osu.ppy.sh/users/2703742 "RedLeaf") |
+| ![][flag_TH] | **Thailand** | **[FrostxE](https://osu.ppy.sh/users/199669 "FrostxE")**, [- Phantasma -](https://osu.ppy.sh/users/1427407 "- Phantasma -"), [bossm](https://osu.ppy.sh/users/654123 "bossm"), [Mikkuri](https://osu.ppy.sh/users/317494 "Mikkuri"), [NonxE](https://osu.ppy.sh/users/319312 "NonxE"), [Romantic](https://osu.ppy.sh/users/1592894 "Romantic") |
+| ![][flag_UA] | **Ukraine** | **[Aka](https://osu.ppy.sh/users/1307553 "Aka")**, [wasisdasS](https://osu.ppy.sh/users/1999698 "wasisdasS"), [FllareA](https://osu.ppy.sh/users/1163931 "FllareA"), [Granje](https://osu.ppy.sh/users/496387 "Granje"), [-Ranndom-](https://osu.ppy.sh/users/5022536 "-Ranndom-"), [Yoru-hide](https://osu.ppy.sh/users/791121 "Yoru-hide"), [blednak](https://osu.ppy.sh/users/912627 "blednak") |
+| ![][flag_GB] | **United Kingdom** | **[jesus1412](https://osu.ppy.sh/users/230116 "jesus1412")**, [Doomsday](https://osu.ppy.sh/users/18983 "Doomsday"), [Tasty Beverage](https://osu.ppy.sh/users/960620 "Tasty Beverage"), [Xim](https://osu.ppy.sh/users/2083664 "Xim"), [Raiku](https://osu.ppy.sh/users/1525538 "Raiku"), [Bubbleman](https://osu.ppy.sh/users/5182050 "Bubbleman"), [Run-Cat](https://osu.ppy.sh/users/4361729 "Run-Cat") |
+| ![][flag_US] | **United States** | **[Xilver](https://osu.ppy.sh/users/3099689 "Xilver")**, [Natalia](https://osu.ppy.sh/users/2162669 "Natalia"), [Axarious](https://osu.ppy.sh/users/2614511 "Axarious"), [Ritzeh](https://osu.ppy.sh/users/1028387 "Ritzeh"), [Seouless](https://osu.ppy.sh/users/3328676 "Seouless"), [0120](https://osu.ppy.sh/users/1901534 "0120"), [Zodiaack](https://osu.ppy.sh/users/3096229 "Zodiaack"), [[Toy]](https://osu.ppy.sh/users/2757689 "[Toy]") |
+
+
+## Groups 
+
+| Group A | Group B | Group C | Group D | Group E | Group F | Group G | Group H |
```

Comment:
When groups has info about the different seeds, its prefered to use the next format:
```
| Group | Top Seed | High Seed | Low Seed | Unseeded | (Other seeds if there are) ... |
| **A** | [Flag] Country name | ... | ... | ... | ... | 
...
```
In this case information about seeds is here
![OWC 2015 Seeds](https://walter.s-ul.eu/PUY3lWHh.png)